From eaa3945f4c2c9c88edb372b8f4c8afc26aa6772c Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Thu, 6 Feb 2020 16:21:44 +0200
Subject: [PATCH 2/9] Revert "Ignore unexpected ACTION_DOWN for SIM avatar
 icons"

	modified:   src/com/android/messaging/ui/conversation/SimIconView.java
---
 .../android/messaging/ui/conversation/SimIconView.java   | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/src/com/android/messaging/ui/conversation/SimIconView.java b/src/com/android/messaging/ui/conversation/SimIconView.java
index 6799bad..e2e446c 100644
--- a/src/com/android/messaging/ui/conversation/SimIconView.java
+++ b/src/com/android/messaging/ui/conversation/SimIconView.java
@@ -19,7 +19,6 @@ import android.content.Context;
 import android.graphics.Outline;
 import android.net.Uri;
 import android.util.AttributeSet;
-import android.view.MotionEvent;
 import android.view.View;
 import android.view.ViewOutlineProvider;
 
@@ -44,14 +43,6 @@ public class SimIconView extends ContactIconView {
         }
     }
 
-    @Override
-    public boolean onTouchEvent(MotionEvent event) {
-        if (isClickable()) {
-            return super.onTouchEvent(event);
-        }
-        return true;
-    }
-
     @Override
     protected void maybeInitializeOnClickListener() {
         // TODO: SIM icon view shouldn't consume or handle clicks, but it should if
-- 
2.25.1

